GiniVision/Classes/CameraViewController.swift

```diff
@@ -118,6 +119,16 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
     fileprivate var cameraFocusLarge: UIImage? {
         return UIImageNamedPreferred(named: "cameraFocusLarge")
     }
+    fileprivate let shouldShowFileImportToolTipUserDefaultsKey = "shouldShowFileImportToolTip"
+    fileprivate var shouldShowFileImportToolTip:Bool {
```

Comment:
This class is already overburdened with a lot of code. I'd try to not have additional logic like that. Maybe you should consider moving it somewhere else.

Reply:
I thought about that, I decided first to put it in the ToolTipView class itself but I thought that it should be more generic, so I changed my mind. What do you think/suggest?

Reply:
In my experience, you will never reuse `ToolTipView ` in another project. Also, if you use it in another place, you might still want to only show it once, hence you will still need to keep something in the user defaults. So, I'd be ok with moving that to the `ToolTipView` class.

Alternatively, you should have a dedicated class. For instance, look at `RatingRequirementsStorage` in the gini pay project. However, for the ToolTipView class that's probably an overkill.